eng/common/scripts/stress-testing/deploy-stress-tests.ps1

```diff
@@ -20,7 +20,8 @@ $ErrorActionPreference = 'Stop'
 . $PSScriptRoot/find-all-stress-packages.ps1
 $FailedCommands = New-Object Collections.Generic.List[hashtable]
 
-if (!(Get-Module powershell-yaml)) {
+if (!(Get-Module -ListAvailable powershell-yaml)) {
```

Comment:
FYI you should probably use https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Helpers/PSModule-Helpers.ps1 and use the Install-ModuleIfNotInstalled function.